CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ The changelog for **SwifterSwift**. Also see the [releases](https://github.com/S
 
 ## Upcoming Release
 ### Added
+- Support for Android
```

Comment:
```suggestion
- **Android**
  - Added Android platform support. [#1230](https://github.com/SwifterSwift/SwifterSwift/pull/1230) by [Marc Prud'hommeaux](https://github.com/marcprux)
```

Reply:
Agreed. I've committed the suggested change.